applications/matter_bridge/doc/matter_bridge_description.rst

```diff
@@ -572,6 +572,26 @@ The application supports the following configurations:
 
        Enables only the necessary application functionalities to optimize its performance.
 
+Configure the functionality of the Matter-Bridge device
```

Comment:
@annwoj  probably can give you more ideas of the documentation structure, but from my perspective I would suggest the following things:
- add the brief description telling about optional functionality that is enabling smart plug device type. It's worth to mention there that it present we can integrate bridge functionality in a real device which is smart plug and do not have to create a bridge device as a standalone device (such as router or Wi-Fi AP). I think it should be somewhere high on the page. Maybe as a subsection of `Overview` section.
- add the description how to enable this functionality using west command (what you've already done). I would suggest moving it into `Building and running` section as a separate subsection. Also you don;t have to describe how to build basic configuration, as it's already done in other place.
- update the `User interface` section to inform that LED2 and BUTTON2 are used in case of the smart plug variant.

Reply:
For now I added some small suggestions, but agree with all the structure-related points above. Also, omitted the "basic functionality" part, as it seems it is covered